js/compiler.ts

```diff
@@ -469,6 +469,31 @@ export class DenoCompiler
     return moduleMetaData.outputCode;
   }
 
+  /**
+   * TODO:
+   * Incremental compilation of code used in REPL.
+   */
+  incrementalCompile(
+    sourceCode: SourceCode,
+    previousOutput?: OutputCode
+  ): { diagnostics: ts.Diagnostic[], outputCode: OutputCode, additionalCode: OutputCode } {
+        
+    console.log('incrementalCompile', sourceCode, previousOutput);
+      
+    // const service = this._service;
+
+    // TODO:
+    // 1. transpile code to JS
```

Comment:
My idea was to start here with `const output = ts.transpileModule(sourceCode, {});` but I'm having trouble running that